input/examples/patient-fred-roberts.xml

```diff
@@ -6,7 +6,7 @@
         <valueString value="Patient - Fred Roberts"/>
     </extension>
     <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-        <valueMarkdown value="Fred Roberts"/>
+        <valueMarkdown value="This is a patient example using the *AU Core Patient* profile."/>
```

Comment:
Insufficient, provide information on the patient example. This project task isn't about better grammar it's about layout information for the reader.